### PR TITLE
 VLCAlertViewController: VLCAlertButton: Add style parameter

### PR DESCRIPTION
**Pull request by Mikanbu**
### Checklist
- [x] I've run `bundle exec fastlane test` from the root directory to see all new and existing tests pass
- [x] I've followed the [vlc-ios code style](Docs/CodingStyle.md) and run `bundle exec fastlane lint` to ensure the code style is valid
- [x] I've read the [Contribution Guidelines](https://github.com/videolan/vlc-ios#contribute)
- [x] I've updated the documentation if necessary.

### Description

Minor addition of `style` parameter to `VLCAlertButton`_(change need for my medialibrary branch)._ 😄
Additionally a bit of cosmetic changes.

### CODE REVIEW

File: Sources/LocalNetworkConnectivity/VLCNetworkServerBrowserViewController.m [hunk 1]
```diff
@@ -85,7 +85,8 @@ - (void)networkServerBrowser:(id<VLCNetworkServerBrowser>)networkBrowser request
                                          errorMessage:NSLocalizedString(@"LOCAL_SERVER_CONNECTION_FAILED_MESSAGE", nil)
                                        viewController:self
                                         buttonsAction:@[[[VLCAlertButton alloc] initWithTitle: NSLocalizedString(@"BUTTON_OK", nil)
-                                               action: ^(UIAlertAction* action){}]]];
+                                                                                        style: UIAlertActionStyleDefault
```
Comment (carolanitz) [inline Sources/LocalNetworkConnectivity/VLCNetworkServerBrowserViewController.m]: you shouldn't need that with the default param right?

File: Sources/VLCBoxTableViewController.m [hunk 2]
```diff
@@ -233,14 +233,16 @@ - (void)triggerDownloadForCell:(VLCCloudStorageTableViewCell *)cell
...
+                                                                                     style: UIAlertActionStyleCancel
+                                                                                    action: ^(UIAlertAction* action){
+                                                                                        self->_selectedFile = nil;
+                                                                                    }],
                                                      [[VLCAlertButton alloc] initWithTitle:NSLocalizedString(@"BUTTON_DOWNLOAD", nil)
-                                                                                  action:^(UIAlertAction* action){
-                                                                                      [self->_boxController downloadFileToDocumentFolder:self->_selectedFile];
-                                                                                      self->_selectedFile = nil;
-                                                                                  }]];
+                                                                                     style: UIAlertActionStyleDefault
```
Comment (carolanitz) [inline Sources/VLCBoxTableViewController.m]: same here

File: Sources/LocalNetworkConnectivity/VLCNetworkServerBrowserViewController.m (hunk 1 above)
Comment (Mikanbu) [reply Sources/LocalNetworkConnectivity/VLCNetworkServerBrowserViewController.m]: Sadly, I believe it's not possible in Objective-c. 😢

**Comment by carolanitz** [comment]
you could add a convenience init and mark at as ios only ```convenience init(title: String, action: AlertAction? = nil) {
        self.init(title: title, style: .default, action: action)
    }```

**Pull request merged**